region/maridia/inner-green/East Sand Hall.json

```diff
@@ -1757,7 +1882,72 @@
           ]}
         ]}
       ],
+      "flashSuitChecked": true,
       "note": "To save Energy, spark diagonally to get onto the pillar, then jump to the door."
+    },
+    {
+      "link": [5, 1],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 53, "excessFrames": 8}}
+      ],
+      "flashSuitChecked": true,
+      "devNote": [
+        "With a controlled shinespark end, starting with 74 energy, this can collect an Evir drop."
+      ]
+    },
+    {
+      "link": [5, 2],
+      "name": "Base",
+      "requires": [],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [5, 4],
+      "name": "Base",
+      "requires": [
+        "Gravity"
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "id": 42,
+      "link": [5, 4],
+      "name": "Suitless In Sand",
+      "requires": [
+        "canSuitlessMaridia",
+        "HiJump",
+        "canPlayInSand"
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [5, 4],
+      "name": "Frozen Evir",
+      "requires": [
+        {"obstaclesNotCleared": ["A"]},
+        "canTrickyUseFrozenEnemies",
+        "canTrickyJump"
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Use the Evir as a platform, which should have been frozen soon after entering the room."
+      ]
+    },
+    {
+      "link": [5, 4],
+      "name": "Power Bomb and Freeze Evir",
+      "requires": [
+        "h_usePowerBomb",
+        "canTrickyUseFrozenEnemies",
+        "canPlayInSand"
```

Comment:
Not seeing why this one has playinsand and the one above doesn't. If anything, this one I would think you can choose your freezing position.

With having to place a PB, I wonder if it needs a tricky jump or evir projectile hit